keps/sig-scheduling/3094-pod-topology-spread-considering-taints/README.md

```diff
@@ -509,6 +508,9 @@ enhancement:
   CRI or CNI may require updating that component before the kubelet.
 -->
 
+If we have a `V+1` version of api-server(`V` refers to the kubernetes version we support this feature), and a `V` version scheduler,
+we'll just ignore `NodeInclusionPolicy` with no side effects.
+
 ## Production Readiness Review Questionnaire
```

Comment:
Please also update kep.yaml and appropriate file from keps/prod-readiness

Reply:
Updated.